README.md

```diff
@@ -131,6 +139,116 @@ To start either of the interfaces in development mode:
   - Unit-tests and UI-tests for all interfaces; Run with `npm test` in each `interfaces/*`-folder.
   - Unit-tests and integration-tests for all services; Run with `npm test` in each `services/*`-folder.
 
+## Common problems with Local Environment set-up
+If the swagger UI is not accessible after installing docker and setting up services, we can take following steps to debug:
+1. `docker ps` to list running containers and their statuses
+2. `docker logs -f <container-name>` to check their statuses
```

Comment:
```suggestion
1. `docker-compose ps` to list running containers and their statuses
2. `docker-compose logs -f <container-name>` to check their statuses
```

We should use `docker-compose` as that is aware of all the other services in our `docker-compose.yml`-file, and not on the individual containers.